vclib-openid/build.gradle.kts

```diff
@@ -13,25 +13,34 @@ val artifactVersion: String by extra
 group = "at.asitplus.wallet"
 version = artifactVersion
 
+
+val iosDisabled: String? by extra
```

Comment:
Why?

Reply:
to work around an IDEA bug. it is now documented in DEVELOPMENT.md